code/app.js

```diff
@@ -37,6 +37,9 @@ app.use(passport.session());
 
 // connect routers
 app.use('/api/user/auth', require('./routers/user/auth'));
+app.use('/api/user/msg', require('./routers/user/msg'));
```

Comment:
- I think we could possibly move the contact us (msg) api out of the user api calls. What do you think?
    - (ie: change `/api/user/msg` → `/api/msg`)
- Also what do you think about changing it from `msg` to `contact-us`?

Reply:
I like that. I'll modify it.

Reply:
I've done this instead:
app.use('/api/user/msg', require('./routers/contact_us/msg'));

Reply:
Looks awesome. One last thing is to make `/api/user/msg` reflect the new location. So changing it to just `/api/msg`

Reply:
Ok, It's done.